src/sentry/api/endpoints/organization_events_trendsv2.py

```diff
@@ -94,6 +96,18 @@ def get(self, request: Request, organization) -> Response:
         except NoProjects:
             return Response([])
 
+        modified_params = params.copy()
+        delta = modified_params["end"] - modified_params["start"]
+        duration = delta.total_seconds()
+        if duration >= 1209600 and duration <= 1209600 * 2:
```

Comment:
Just a nit, 1209600 is 14 days in seconds right? 1209600 * 2 then isn't completely 30 days, do we want to make it 30 days?

Reply:
ah yeah good catch. I realized it too lol. I'll fix it properly once we validate the approach. sorry just don't want to wait for ci again